README.md

```diff
@@ -2,7 +2,19 @@
 
 # bitwarden Help Center
 
-The bitwarden help center project is written using Jekyll, which is hosted/deployed to a GitHub pages branch.
+The bitwarden help center project is written using Jekyll, which is hosted/deployed to a GitHub pages branch ([https://help.bitwarden.com/](https://help.bitwarden.com/)).
+
+# Build/Run
+**Requirements**
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Jekyll](https://jekyllrb.com/)
+
+Run the following commands:
+- `jekyll build`
+- `jekyll serve`
+
+You can now access the help center at `localhost:4009`.
```

Comment:
Why is this showing as a change? Weird. Do you need to merge upstream? https://help.github.com/articles/syncing-a-fork/